src/library/librarycontrol.h

```diff
@@ -1,6 +1,7 @@
 #ifndef LIBRARYMIDICONTROL_H
 #define LIBRARYMIDICONTROL_H
 
+#include <memory>
```

Comment:
Heh, nice -- we usually have to take it slow due to Microsoft, though we're almost ready to switch to VS2015 which would bring a lot of c++14 features:
https://blogs.msdn.microsoft.com/vcblog/2015/06/19/c111417-features-in-vs-2015-rtm/